src/ASBNApp.DataAPI/Program.cs

```diff
@@ -0,0 +1,74 @@
+using ASBNApp.DataAPI.Context;
+using ASBNApp.DataAPI.Models;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.OData;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.OData.ModelBuilder;
+
+
+var builder = WebApplication.CreateBuilder(args);
+
+// Identity stuff
+builder.Services.AddAuthentication(IdentityConstants.ApplicationScheme).AddIdentityCookies();
+builder.Services.AddAuthorizationBuilder();
+builder.Services.AddIdentityCore<User>()
+	.AddEntityFrameworkStores<ASBNAppContext>()
+	.AddApiEndpoints();
+
+// Add services to the container.
+// Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
+builder.Services.AddEndpointsApiExplorer();
+builder.Services.AddSwaggerGen();
+
+// Setting up the database
+var config = new ConfigurationBuilder()
+    .SetBasePath(AppDomain.CurrentDomain.BaseDirectory)
+    .AddJsonFile("appsettings.json")
+    .Build();
+
+// Configuring CORS
+builder.Services.AddCors(options =>
+{
+    options.AddPolicy("AllowASBNAppFrontend", builder =>
+    {
+        builder.WithOrigins("https://localhost:5227")
+			.AllowAnyHeader()
+            .AllowAnyMethod()
+            .AllowCredentials();
+    });
+});
+
+// Create the EDM models
+var modelBuilder = new ODataConventionModelBuilder();
+modelBuilder.EntitySet<User>("User");
+modelBuilder.EntitySet<Entry>("Entry");
+modelBuilder.EntitySet<WorkLocation>("WorkLocation");
+
+builder.Services.AddControllers().AddOData(
+    options => options.Select().Filter().OrderBy().Expand().Count().SetMaxTop(null).AddRouteComponents(
+        "odata",
+        modelBuilder.GetEdmModel()));
+
+builder.Services.AddDbContext<ASBNAppContext>(
+    options => options.UseSqlServer(config.GetConnectionString("Default")));
+
+// Finalizing
+var app = builder.Build();
+
+// Set CORS policy
+app.UseCors("AllowASBNAppFrontend");
```

Comment:
use a constant for magic strings ;)

Reply:
You mean like `const string frontendPolicy = "AllowASBNAppFrontend"`? Is that safer because it can't be changed during runtime?